6_temp_coop_fetch/out/coop_all_files.rds.tind

```diff
@@ -1,3 +1,8 @@
+<<<<<<< Updated upstream
 hash: 8ae5ea9c8dbf3d39ca7999d9bfe92be9
 freshened: 2022-02-16 15:06:03 -0600
```

Comment:
I have an accidental merge conflict hanging here because I was accidentally working on the `main` branch instead of another branch. I don't have much experience resolving these yet and out like some advice on how best to handle this.

Reply:
Delete everything in that file except for 
```yaml
hash: df810ac9084dc0757a79d40ab70f5c7a
freshened: 2022-02-16 16:46:32 -0500
```
And leave the return line. The conflict text shows you your version (which you should keep and commit here) and the version that conflicts with it. This way you manually choose which to keep.

Reply:
Done! Thank you!